lib/money/money.rb

```diff
@@ -28,6 +30,7 @@ class UndefinedSmallestDenomination < StandardError; end
   def cents
     fractional
   end
+  alias_method :pence, :cents
```

Comment:
Okay, I've removed it and rebased.

Why remove cents though? It seems useful to me as (I imagine) that the Money gem will be used with USD (and CAD, AUD, NZD, several other currencies which use 'cents') very frequently and this could make code more readable.

That being said, I think the name 'cents' is kinda ambiguous. If you told me to give you the "cents" from "$12.34", I wouldn't be sure if you meant "1234 cents" or "34 cents" (i.e. just the part after the decimal point.) A more clear name might be `in_cents`.